cmake_format/vscode_extension/src/extension.ts

```diff
@@ -16,31 +19,17 @@ export function activate(context: vscode.ExtensionContext) {
   let disposable = vscode.languages.registerDocumentFormattingEditProvider(
     { scheme: 'file', language: 'cmake' }, {
     provideDocumentFormattingEdits(document: vscode.TextDocument): vscode.TextEdit[] {
-      var fs = require('fs')
-      var path = require('path')
       var config = vscode.workspace.getConfiguration('cmakeFormat');
-      var exePath = config.get("exePath");
+      var exePath = config.get<string>("exePath", "cmake-format");
```

Comment:
Default is defined [here][1]. It should not be defined in two places. 

[1]: https://github.com/cheshirekow/cmake_format/blob/master/cmake_format/vscode_extension/package.json#L60

Reply:
How can I guarantee that `config.get<string>("exePath")` does not return `undefined` This is a potential error otherwise.

Reply:
Hm.. I'm not sure. Presumably there is documentation on the vscode config system somewhere.  My expectation is that returning undefined is not possible if the config has a default value (which it does).

Reply:
Yeah, I agree that it shouldn't return `undefined` given the configuration in `package.json`. However, that method can still return `undefined`

See documentation [here](https://code.visualstudio.com/api/references/vscode-api):
> get<T>(section: string): T | undefined

My thought process for making that change was that the rest of the code expects that to *not* be `undefined` so I added the default to make that guarantee something that the syntax checker could recognize.

Reply:
I see. That's not a bad thing in general, but I think it's a bad choice here since it means specifying the default in two places. If it is undefined, the error report will be a bit obtuse, but still a visible error. Also, it indicates a problem in the default config so should likely be visible.